### PR TITLE
Add resources option

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: helm/k8s-initiator-app/values.yaml [hunk 1]
```diff
@@ -7,6 +7,12 @@ command:
   - |
     echo "No command set"
 
+# container resources
+resources: {}
+
+# Make room to run the pod
+priority_class: system-node-critical
```
Comment (omegas27) [inline helm/k8s-initiator-app/values.yaml:14]: I think the helm prefer to use priorityClassName instead

**Comment by omegas27** [comment]
You just made my day with the part about setting the OIDC <3

**Comment by pipo02mix** [comment]
Corrected PTAL

> You just made my day with the part about setting the OIDC <3

Indeed was added by Sir Rosello

**Comment by T-Kukawka** [comment]
awesome stuff 🚀

**Comment by omegas27** [comment]
FYI: Setting that priority class makes it only installable in kube-system

**Pull request merged**